description.md

```diff
@@ -40,6 +40,61 @@ Put differently, it shows how each content type is to be used in the various tea
 
 ### Self-Study
 
+Self-Study is the process of independently learning outside of structural classroom settings or formal instruction.
+
+There are multiple reasons students find self-studying hard:
+
+- **Lack of structure**: Learning without a structure can lead to potential confusion and inefficiency.
+- **Access to valuable resources**: Even if there are a lot of resources on the internet, they are not always well structured.
+- **Lack of feedback**: Without regular feedback from the teacher, students tend to lose motivation, potentially leading to gaps in knowledge.
+- **Need for help**: Without help from the teacher, students may not be able to solve the problems on their own and this can lead to frustration.
+- **Difficulty**: The subjects must have an adequate difficulty in order not to demotivate students.
+
+In order to overcome these problems, we can improve the following:
+
+#### Organize resources
+
+The material is to be organized into distinct chapters.
+Each chapter will encompass one or more subjects.
+Within each chapter, there will be an `overview` section that summarizes the different subjects included.
+This `overview` will highlight the main topics of the chapter and the key information that readers should grasp before moving on to the next chapter.
+A general example of the following structure is shown below or can be found [here](https://github.com/open-education-hub/oer-template).
+
+```bash
+└── chapters/
+    ├── chapter-1/
+    │   ├── overview
```

Comment:
```suggestion
    │   ├── overview.md
```

Reply:
@gabrielmocanu , I'm leaving this, but note that it shouldn't be here. It's content-centric, when it should be content delivery-centric.